airflow/providers/google/cloud/sensors/gcs.py

```diff
@@ -226,14 +228,15 @@ class GCSUploadSessionCompleteSensor(BaseSensorOperator):
 
     template_fields = ('bucket', 'prefix')
     ui_color = '#f0eee4'
+    previous_objects: Set[str]
 
     @apply_defaults
     def __init__(self,
                  bucket: str,
                  prefix: str,
                  inactivity_period: float = 60 * 60,
                  min_objects: int = 1,
-                 previous_num_objects: int = 0,
+                 previous_objects: Optional[Set[str]] = None,
```

Comment:
This is a change in the signature that should need a note in Updating.md

Reply:
done.